source/apiVolontaria/volunteer/admin.py

```diff
@@ -88,6 +92,53 @@ class EventAdmin(admin.ModelAdmin):
     ordering = ('start_date',)
 
 
+class CycleAdmin(admin.ModelAdmin):
+
+    list_display = [
+        'name',
+        'start_date',
+        'end_date',
+    ]
+
+    date_hierarchy = 'start_date'
+
+    actions = ['generate_participation_report_csv', ]
+
+    def generate_participation_report_csv(self, request, queryset):
```

Comment:
Can you add a documentation on this function since it's not really standard for the moment ?
```
def generate_participation_report_csv(self, request, queryset):
    """
    Little description
    :param request: blahblahblah 
    :param queryset: blahblahblah
    :return: the format/structure of the reponse you return
    """
```

Reply:
Good Point